docs/dependency-injection.md

```diff
@@ -27,6 +27,7 @@ The code that is responsible for providing the dependencies a class needs is usu
 
 create() method
 ------------------
+:octicons-tag-24: 12.0+
```

Comment:
This is backported to Drush 11, but not yet released; maybe this should be 11.6+?